core/include/detray/definitions/math.hpp

```diff
@@ -24,4 +24,36 @@ namespace math_ns = cl::sycl;
 namespace math_ns = std;
 #endif  // SYCL
 
-}  // namespace detray
\ No newline at end of file
+namespace detail {
+
+using math_ns::copysign;
+using math_ns::signbit;
+
+/// Composes a floating point value with the magnitude of @param mag and the
```

Comment:
leftovers?

Reply:
Actually, I would like to keep those. The CUDA test fail if I use the CUDA intrinsics and I have not figured out why, yet